doc/getting-started/apl-nuc.rst

```diff
@@ -47,41 +47,50 @@ complete this setup.
    and follow the `Clear Linux installation guide
    <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install>`__
    as a starting point for installing Clear Linux onto your platform.  Follow the recommended
-   options for choosing an **Automatic** installation type, and using the platform's
+   options for choosing an **Manual** installation type, and using the platform's
    storage as the target device for installation (overwriting the existing data
    and creating three partitions on the platform's storage drive).
+   
+   High-level steps should be: 
+   
+   #.  Install Clear on a NUC Use the "Manual" option.
+   #.  Use default partition scheme for storage
+   #.  Name the host "clr-sos-guest"
+   #.  Add an administrative user "clear" with "sudoers" privilege
+   #.  Add these additional bundles "editors", "user-basic", "desktop-autostart", "network-basic"
+   #.  For network, choose “DHCP”
 
 #. After installation is complete, boot into Clear Linux, login as
-   **root**, and set a password.
+   **clear**, and set a password.
 
 #. Clear Linux is set to automatically update itself. We recommend that you disable
    this feature to have more control over when the updates happen. Use this command
    to disable the autoupdate feature:
 
    .. code-block:: none
 
-      # swupd autoupdate --disable
+      # sudo swupd autoupdate --disable
```

Comment:
The ``#`` prompt indicates you're already an admin, so change the prompt to ``$`` (or leave off the sudo).